lib/utils/utils.js

```diff
@@ -270,3 +275,62 @@ function findUnorderedProperty(arr) {
 
   return null;
 }
+
+/**
+ * Checks whether a computed property has duplicate dependent keys.
+ *
+ * @param  {CallExpression} callExp Given call expression
+ * @return {Boolean}        Flag whether dependent keys present.
+ */
+function hasDuplicateDependentKeys(callExp) {
+  if (!ember.isComputedProp(callExp)) return false;
+
+  const dependentKeys = parseDependentKeys(callExp);
+  const uniqueKeys = dependentKeys
+    .filter((val, index, self) => self.indexOf(val) === index);
+  return uniqueKeys.length !== dependentKeys.length;
+}
+
+/**
+ * Parses dependent keys from call expression and returns them in an array.
+ *
+ * It also unwraps the expressions, so that `model.{foo,bar}` becomes `model.foo, model.bar`.
+ *
+ * @param  {CallExpression} callExp CallExpression to examine
+ * @return {Array}          Array of unwrapped dependent keys
+ */
+function parseDependentKeys(callExp) {
+  const isMemberExpCallExp = callExp.arguments.length === 0 && callExp.callee.object;
```

Comment:
I'd add comment above why do we check this. I was confused at first and then I saw a case with `.volatile` in tests.

You should also check it rather like so:
```
!callExp.arguments.length &&
isMemberExpression(callExp.callee) &&
isCallExpression(callExp.callee.object)
```